src/Avalonia.Controls/TextBlock.cs

```diff
@@ -198,10 +197,12 @@ public Thickness Padding
         /// </summary>
         public string? Text
         {
-            get => GetText();
-            set => SetText(value);
+            get => GetValue(TextProperty);
+            set => SetValue(TextProperty, value);
         }
 
+        private string? DebugText => Text ?? Inlines?.Text;
```

Comment:
This can be removed one can always inspect Inlines.Text for complex text

Reply:
This property is referenced by the `DebuggerDisplay` attribute at the top of the class, but it's embedded into a string so hard to detect. I'll switch to `nameof` so that the IDE can detect the reference.